### ISSUE TITLE
Console output not displaying after first action

### ISSUE THREAD

**Issue by leoafarias** [leoafarias/sidekick#138]
## Problem
The console output at the bottom of Sidekick displays correctly after the first action on a release. An action could be `install, remove, set as global, setup`. 

However, after a second action takes place the loading indicator and feedback are displayed however no console information is displayed.

The error displayed in the logs is.
```
Unhandled Exception: Bad state: Stream has already been listened to.
```

This seems to be coming from `fvmStdoutProvider` on fvm.provider.dart, this had been working previously but there seems to be a regression.

I believe it's time to review the console log and consider a refactor on this display. Since then the sandbox feature is done and also works more efficiently with better UX.

**Comment by aguilaair** [comment]
I can try working on this but feel that you have a better knowledge of how the console works (having designed it and worked on sandbox previously)

**Comment by aguilaair** [comment]
The core bug I can fix, I think it should be a separate issue

**Comment by leoafarias** [comment]
@aguilaair ok I changed the issue to focus on just the bug itself, and will do a refactor later

**Comment by aguilaair** [comment]
Ok, I've been checking this out for the last few days and there seems to be more than meets the eye. The initial bug is easily solved, just by ensuring that the stream provider is not disposed, for example by not closing the bottom app bar, or by removing the `autodispose` parameter from `fvmStdoutProvider`. 

However, then we can see a second bug that is caused by the merging of the stdOut, Err, etc.

`Bad state: Cannot add event while adding a stream`
```
[ERROR:flutter/lib/ui/ui_dart_state.cc(209)] Unhandled Exception: Bad state: Cannot add event while adding a stream
#0      _StreamController.add (dart:async/stream_controller.dart:553:24)
#1      _StreamSinkWrapper.add (dart:async/stream_controller.dart:826:13)
#2      runExecutableArguments.<anonymous closure>
package:process_run/src/process_run.dart:147
#3      runExecutableArguments.<anonymous closure>
package:process_run/src/process_run.dart:145
#4      _rootRunUnary (dart:async/zone.dart:1436:47)
#5      _CustomZone.runUnary (dart:async/zone.dart:1335:19)

```
This bug appears after 2 or 3 processed have finished, and I am still looking for the reason. I will keep the tread updated.

**Comment by leoafarias** [comment]
Thanks @aguilaair let me know if you need assistance

**Comment by leoafarias** [comment]
I believe this has been closed with #159
